src/System.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -74,6 +74,20 @@ public static bool IsLocalDBInstalled()
             return false;
         }
 
+        public static bool IsIntegratedSecuritySetup()
+        {
+            string integratedSecuritySetupFlag = Environment.GetEnvironmentVariable("TEST_INTEGRATEDSECURITY_SETUP");
+            if (!string.IsNullOrWhiteSpace(integratedSecuritySetupFlag))
```

Comment:
This check can be avoided. 
int.TryParse(null, out int result) => returns False;
int.TryParse(" ", out int result) => returns False;
int.TryParse("", out int result) => returns False;